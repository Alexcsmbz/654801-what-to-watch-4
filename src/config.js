export const moviePageTabs = [
  {
    name: `Overview`,
  },
  {
    name: `Details`,
  },
  {
    name: `Reviews`,
  },
];

export const RoutePath = {
  MAIN: `/`,
  LOGIN: `/login`,
  MOVIE: `/films/:id`,
  MYLIST: `/mylist`,
  REVIEW: `/films/:id/review`,
};
