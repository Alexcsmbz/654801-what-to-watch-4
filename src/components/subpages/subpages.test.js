import Subpages from './subpages.jsx';
import Overview from 'components/overview/overview.jsx';
import Details from 'components/details/details.jsx';
import Reviews from 'components/reviews/reviews.jsx';
import {testMovie} from 'config';

describe(`Subpages snapshot test`, () => {
  it(`Subpages should render subpages`, () => {
    const tree = renderer.create(
        <Subpages idx={0}>
          <Overview movie={testMovie} />
          <Details movie={testMovie} />
          <Reviews commentList={[]} />
        </Subpages>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
