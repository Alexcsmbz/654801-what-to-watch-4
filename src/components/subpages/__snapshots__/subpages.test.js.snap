// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subpages snapshot test Subpages should render subpages 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8.8
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        370881
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      description
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        and other...
      </strong>
    </p>
  </div>,
]
`;
